lib/src/cupertino_settings_item.dart

```diff
@@ -40,6 +43,9 @@ class CupertinoSettingsItem extends StatefulWidget {
   final int subtitleMaxLines;
   final Widget leading;
   final Widget trailing;
+  final IconData iosChevron;
+  final Color iosChevronColor;
+  final double iosChevronSize;
```

Comment:
Wouldn't it be easier to just allow an `Icon` widget instead of specifying the various properties and constructing the icon?
That way the user will have more control than these properties we've given.

Reply:
Agreed! I'll update the pr.